DetectorDescription/DDCMS/interface/DDDetector.h

```diff
@@ -34,6 +35,8 @@ namespace cms {
 
     dd4hep::Detector const* description() const { return m_description; }
 
+    DDParsingContext* m_context;
```

Comment:
@cvuosalo - `DDDetector` should not have `DDParsingContext`. The `DDParsingContext` is created for a `DDDetector`:
```c++
cms::DDParsingContext context(det);
```